internal/gatewayapi/clienttrafficpolicy.go

```diff
@@ -644,3 +649,39 @@ func (t *Translator) translateListenerTLSParameters(policy *egv1a1.ClientTraffic
 
 	return nil
 }
+
+func translateListenerConnection(connection *egv1a1.Connection, httpIR *ir.HTTPListener) error {
+	// Return early if not set
+	if connection == nil {
+		return nil
+	}
+
+	irConnection := &ir.Connection{}
+
+	if connection.Limit != nil {
+		hasLimit := false
```

Comment:
so `hasLimit` is being used to make sure at least one of the sub fields is set ? 
isn't this check already happening in CEL ?